omnibus/config/software/elasticsearch.rb

```diff
@@ -25,14 +25,14 @@
 
 relative_path "elasticsearch-#{version}"
 
-version "6.8.18" do
+version "6.8.20" do
```

Comment:
Since we only use one and this is in the server you might as well nuke .20

Reply:
Done.